service/test/controllers/BaseControllerTest.java

```diff
@@ -39,12 +31,18 @@ public BaseControllerTest() {
   }
 
   public void baseControllerTestsetUp() {
-
-    application = PowerMockito.mock(org.sunbird.Application.class);
-    PowerMockito.mockStatic(org.sunbird.Application.class);
-    PowerMockito.when(org.sunbird.Application.getInstance()).thenReturn(application);
-    application.init();
-    mockRequestHandler();
+    try{
+      application = PowerMockito.mock(org.sunbird.Application.class);
+      PowerMockito.mockStatic(org.sunbird.Application.class);
+      PowerMockito.when(org.sunbird.Application.getInstance()).thenReturn(application);
+      application.init();
+      mockRequestHandler();
+      PowerMockito.mockStatic(ElasticSearchUtil.class);
+      PowerMockito.doNothing().when(ElasticSearchUtil.class, "initialiseESClient", Mockito.anyString(), Mockito.anyString());
```

Comment:
This is fine, try if verify() works in order to check this method or inner method which is actually called or not.

Reply:
@anmol2302 since it is not actual test, no need to value my previous comment.